src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Nightly.Release.nuspec

```diff
@@ -14,6 +14,7 @@
     <summary>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. </summary>
     <description>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. Targets .NET 4.5 and .NET Platform Standard 1.1. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
 OData .NET library is open source at http://github.com/OData/odata.net</description>
+    <copyright>© Microsoft Corporation.  All rights reserved.</copyright>

```

Comment:
I'm guessing this works as opposed to using the escape sequence, i.e. &, then #, then 169, then ;

Reply:
correct.

---
In reply to: [151803293](https://github.com/OData/odata.net/pull/999#discussion_r151803293) [](ancestors = 151803293)

Reply:
As discussed in Rob's office, proceed with the merge.

Reply:
This merged PR has been reverted since the target branch was the release branch.